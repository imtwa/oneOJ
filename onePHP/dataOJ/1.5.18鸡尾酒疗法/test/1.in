14
3740 3489
1905 7879
9709 3949
6438 4282
7545 923
8958 8974
4314 4731
7990 7303
6410 8821
3828 1157
5530 6343
6520 4975
4573 94
9276 8117
