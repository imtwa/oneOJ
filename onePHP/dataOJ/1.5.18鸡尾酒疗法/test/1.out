better
worse
worse
worse
better
better
same
better
worse
better
worse
worse
worse
